src/components/scopes/prose/_prose.scss

```diff
@@ -0,0 +1,126 @@
+.gv-s-prose {
+
+  -webkit-font-smoothing: antialiased;
```

Comment:
Remove this as it’s [set on body](https://github.com/alphagov/govuk_frontend_alpha/blob/02d0a844e28be55f6f303457277d977134af2a94/src/assets/scss/elements/_page.scss#L49)? It’s also [set on button](https://github.com/alphagov/govuk_frontend_alpha/blob/ef5fcaa2c4eed336cf4972c70f620849bbe144eb/src/components/button/_button.scss#L25).